src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -110,11 +105,28 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
                 "jenkins", "node", "master", "computer",
                 "item", "label", "view", "all", "unknown",
                 "user", "anonymous", "authenticated",
-                "everyone", "system", "admin",
-                Jenkins.VERSION
+                "everyone", "system", "admin", Jenkins.VERSION
         ));
     }
 
+    /**
+     * To avoid corrupt the content of the files in the bundle just in case we have an object name as 'a' or '.', we
+     * avoid replacing one single characteres (ascii codes actually). A one single character in other languages could
+     * have a meaning, so we remain replacing them. Example: 日 (Sun)
+     * @return Set of characters in ascii code chart
+     */
+    private static Set<String> getAllAsciiCharacters() {
+        final int SPACE = ' '; //20
+        final int TILDE = '~'; //126
+        Set<String> singleChars = new HashSet<>(TILDE - SPACE + 1);
+
+        for (int i = SPACE; i <= TILDE; i++) {
```

Comment:
```suggestion
        for (char i = SPACE; i <= TILDE; i++) {
```

Can you do that? Can remove need for casting below, and can make SPACE and TILDE chars.